packages/transaction-controller/src/types.ts

```diff
@@ -966,3 +974,68 @@ export type SecurityAlertResponse = {
   result_type: string;
   providerRequestsCount?: Record<string, number>;
 };
+
+/** Gas fee estimates for a specific priority level. */
+export type GasFeeEstimatesLevel = {
```

Comment:
I noticed the terms 'GasFeeEstimatesLevel' and 'GasFeeEstimateLevel' are super close and could mix us up. 

How about renaming this to `GasFeeEstimatesForLevel` or `GasEstimatesTier` or similar?